github_issue_summarization/pachyderm_seldon_kvc/README.md

```diff
@@ -0,0 +1,295 @@
+  ```
+
+Once you have that secret, you can create the build and export pipelines as follows:
+
+```
+$ pachctl create-pipeline -f build.json
+$ pachctl create-pipeline -f export.json
+```
+
+Again, Pachyderm will automatically run the build and export jobs and version any associated data:
+
+```
+$ pachctl list-job
+ID                               OUTPUT COMMIT                                STARTED           DURATION   RESTART PROGRESS  DL       UL       STATE
+b496e8c500df43d58d667d113ee8aee7 export/98c3f65fcd3d4f41bebff0e83452f7db      About an hour ago 15 minutes 0       1 + 0 / 1 27.98MiB 0B       success
+618dc8220d3b433b95dad3e1680f778d build/190f5440fed749d4aec87761f40e9bc6       About an hour ago 53 seconds 0       1 + 0 / 1 27.95MiB 27.98MiB success
+435e6243b4374ba6888d598431200d5d train/a096f70a6ffc43b883285d26dccf4a29       About an hour ago 2 minutes  0       1 + 0 / 1 201KiB   27.75MiB success
+226fec9ba9014032a30660b1b1caff38 pre_process/357232558c0a47eeb9f21ac949c06bb3 About an hour ago 2 minutes  0       1 + 0 / 1 104.2KiB 201KiB   success
+70eb099852b543ce9818bf11dbe68c76 split/99468d94ac51433e8968362b1015fb2c       About an hour ago 2 minutes  0       1 + 0 / 1 2.561MiB 104.2KiB success
+$ pachctl list-repo
+NAME                CREATED             SIZE
+build               About an hour ago   27.98MiB
+export              About an hour ago   0B
+pre_process         About an hour ago   201KiB
+raw_data            About an hour ago   2.561MiB
+split               About an hour ago   104.2KiB
+train               About an hour ago   27.75MiB
+$ pachctl list-file build master build
+NAME                            TYPE                SIZE
+build/Dockerfile                file                778B
+build/Makefile                  file                74B
+build/README.md                 file                367B
+build/body_preprocessor.dpkl    file                113.3KiB
+build/build_image.sh            file                95B
+build/microservice.py           file                6.176KiB
+build/model_microservice.py     file                3.536KiB
+build/output_model.h5           file                27.75MiB
+build/persistence.py            file                1.917KiB
+build/proto                     dir                 76.69KiB
+build/push_image.sh             file                73B
+build/requirements.txt          file                74B
+build/seldon_requirements.txt   file                79B
+build/title_preprocessor.dpkl   file                29.75KiB
+```
+
+You should also be able to see the Seldon model serving image on Docker Hub under:
+
+```
+<your user>/issuesummarization:<pachyderm job id>
+```
+
+That image is now ready to be used in Seldon to server the model. You can follow [this example](https://github.com/kubeflow/examples/tree/master/github_issue_summarization) to deploy Seldon with KubeFlow, deploy the model serving, and deploy an example frontend to query the model.
```

Comment:
This section is somewhat vague about how to complete the walkthrough. More specificity about where to pick up instructions for serving would be helpful.